src/EnergyPlus/ConvectionConstants.hh

```diff
@@ -0,0 +1,180 @@
+int constexpr HcInt_FisherPedersenCeilDiffuserWalls{208};
+int constexpr HcInt_AlamdariHammondStableHorizontal{209};
+int constexpr HcInt_AlamdariHammondVerticalWall{210};
+int constexpr HcInt_AlamdariHammondUnstableHorizontal{211};
+int constexpr HcInt_KhalifaEq3WallAwayFromHeat{212};
+int constexpr HcInt_KhalifaEq4CeilingAwayFromHeat{213};
+int constexpr HcInt_KhalifaEq5WallNearHeat{214};
+int constexpr HcInt_KhalifaEq6NonHeatedWalls{215};
+int constexpr HcInt_KhalifaEq7Ceiling{216};
+int constexpr HcInt_AwbiHattonHeatedFloor{217};
+int constexpr HcInt_AwbiHattonHeatedWall{218};
+int constexpr HcInt_BeausoleilMorrisonMixedAssistingWall{219};
+int constexpr HcInt_BeausoleilMorrisonMixedOppossingWall{220};
+int constexpr HcInt_BeausoleilMorrisonMixedStableCeiling{221};
+int constexpr HcInt_BeausoleilMorrisonMixedUnstableCeiling{222};
+int constexpr HcInt_BeausoleilMorrisonMixedStableFloor{223};
+int constexpr HcInt_BeausoleilMorrisonMixedUnstableFloor{224};
+int constexpr HcInt_FohannoPolidoriVerticalWall{225};
+int constexpr HcInt_KaradagChilledCeiling{226};
+int constexpr HcInt_ISO15099Windows{227};
+int constexpr HcInt_GoldsteinNovoselacCeilingDiffuserWindow{228};
+int constexpr HcInt_GoldsteinNovoselacCeilingDiffuserWalls{229};
+int constexpr HcInt_GoldsteinNovoselacCeilingDiffuserFloor{230};
+
+// parameters for identifying more specific hc model equations, outside face
+int constexpr HcExt_Value{-999};
+int constexpr HcExt_Schedule{-998};
+int constexpr HcExt_ASHRAESimple{1};
+int constexpr HcExt_ASHRAETARP{2};
+int constexpr HcExt_TarpHcOutside{5};
+int constexpr HcExt_MoWiTTHcOutside{6};
+int constexpr HcExt_DOE2HcOutside{7};
+int constexpr HcExt_BLASTHcOutside{8};
+int constexpr HcExt_AdaptiveConvectionAlgorithm{9};
+int constexpr HcExt_None{300}; // none is allowed because Hn and Hf are split
+int constexpr HcExt_UserValue{301};
+int constexpr HcExt_UserSchedule{302};
+int constexpr HcExt_UserCurve{303};
+int constexpr HcExt_ASHRAESimpleCombined{304};
+int constexpr HcExt_NaturalASHRAEVerticalWall{305};
+int constexpr HcExt_NaturalWaltonUnstableHorizontalOrTilt{306};
+int constexpr HcExt_NaturalWaltonStableHorizontalOrTilt{307};
+int constexpr HcExt_SparrowWindward{308};
+int constexpr HcExt_SparrowLeeward{309};
+int constexpr HcExt_MoWiTTWindward{310};
+int constexpr HcExt_MoWiTTLeeward{311};
+int constexpr HcExt_DOE2Windward{312};
+int constexpr HcExt_DOE2Leeward{313};
+int constexpr HcExt_NusseltJurges{314};
+int constexpr HcExt_McAdams{315};
+int constexpr HcExt_Mitchell{316};
+int constexpr HcExt_ClearRoof{317};
+int constexpr HcExt_BlockenWindward{318};
+int constexpr HcExt_EmmelVertical{319};
+int constexpr HcExt_EmmelRoof{320};
+int constexpr HcExt_AlamdariHammondVerticalWall{321};
+int constexpr HcExt_FohannoPolidoriVerticalWall{322};
+int constexpr HcExt_ISO15099Windows{323};
+int constexpr HcExt_AlamdariHammondStableHorizontal{324};
+int constexpr HcExt_AlamdariHammondUnstableHorizontal{325};
```

Comment:
I'm guessing as I read through I'll remember/understand why this list of int constexpr values is not an enum.